specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/dataConnectors.json

```diff
@@ -284,184 +284,169 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/dataConnectorsCheckRequirements": {
```

Comment:
Please call out what APIs/definitions are new/changed in this file.  Looks like it is a combination of an existing file with APIs from another file that existed in a previous version

Reply:
The purpose of this PR is creating a version that contains all our supported endpoints, without introducing any change.
I started this version based of 2021-03-01-preview using OpenAPI Hub and then manually added missing endpoint from 2019-01-01-preview. None of the endpoints are new, but some are missing when compared to 2019 as they aren't supported anymore. The missing endpoints:

- "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/aggregations/{aggregationsName}"
- "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/cases"
- "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/cases/{caseId}"